examples/api/lib/cupertino/date_picker/cupertino_date_picker.0.dart

```diff
@@ -32,14 +32,16 @@ class _DatePickerExampleState extends State<DatePickerExample> {
   DateTime time = DateTime(2016, 5, 10, 22, 35);
   DateTime dateTime = DateTime(2016, 8, 3, 17, 45);
 
-  // This shows a CupertinoModalPopup with a reasonable fixed height which hosts CupertinoDatePicker.
+  // This shows a CupertinoModalPopup with a reasonable fixed height which hosts
```

Comment:
Comments can probably leave out the "This" and there's (arguably) a missing "a":
```dart
// Shows a CupertinoModalPopup with a reasonable fixed height which hosts // a CupertinoDatePicker.
```

Reply:
Well, but without the "This", it's a sentence fragment, not a sentence, and we have a style guide rule that says to use correct grammar, with almost exactly this scenario as an example. I mean, sure, it's talking about capitalization, but it's still incorrect grammar to use sentence fragments.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-correct-grammar

Reply:
It's just as grammatically correct, but it's probably a better sentence. I'll add that.